jenkins/install_jenkins.sh

```diff
@@ -9,8 +10,16 @@ Arguments
   --vm_private_ip|-pi                 : The VM private ip used to configure Jenkins URL. If missing, jenkins_fqdn will be used instead
   --jenkins_release_type|-jrt         : The Jenkins release type (LTS or weekly or verified). By default it's set to LTS
   --jenkins_version_location|-jvl     : Url used to specify the version of Jenkins.
+  --service_principal_type|-sp        : The type of service principal: MSI or manual.
+  --service_principal_id|-sid         : The service principal ID.
+  --service_principal_secret|-ss      : The service principal secret.
+  --subscription_id|-subid            : The subscription ID of the SP.
+  --tenant_id|-tid                    : The tenant id of the SP.
   --artifacts_location|-al            : Url used to reference other scripts/artifacts.
   --sas_token|-st                     : A sas token needed if the artifacts location is private.
+  --cloud_agents|-ca                  : The type of the cloud agents: aci, vm or no.
+  --resource_group|-rg                : the resource group name.
+  --location|-lo                      : the resource group location.
```

Comment:
The resource group and location for cloud agents?

Reply:
for jenkins master, while the agents are configured in the same resource group by default.